README.md

```diff
@@ -0,0 +1,285 @@
+# Duxy
+
+[![npm version](https://img.shields.io/npm/v/duxy.svg)](https://www.npmjs.com/package/duxy) [![License](https://img.shields.io/npm/l/duxy.svg)](https://www.npmjs.com/package/duxy) [![Build Status](https://travis-ci.org/producthunt/duxy.svg)](https://travis-ci.org/producthunt/duxy)
+
+RESTful resources.
+
+## Table of Contents
+
+  1. [Installation](#installation)
+  1. [Usage](#usage)
+    1. [Setup](#setup)
+    1. [Usage](#usage-1)
+    1. [DSL](#dsl)
+      1. [get](#getname--path-)
+      1. [post](#postname--path-)
+      1. [put](putname--path-)
+      1. [patch](#patchname--path-)
+      1. [del](#delname--path-)
+      1. [namespace](#namespacename--path--fn)
+      1. [resources](#resourcesname--path-only-findone-findall-create-update-delete-fn)
+      1. [resource](#resourcename--path-only-findone-create-update-delete--fn)
+    1. [Adapter](#adapter)
+  1. [Development](#development)
```

Comment:
maybe show case a larger duxy file - eg one of producthunt so that people understand how it's awesome